masonry/src/widget/spinner.rs

```diff
@@ -93,15 +93,13 @@ impl Widget for Spinner {
         match event {
             LifeCycle::WidgetAdded => {
                 ctx.request_anim_frame();
-                ctx.request_paint();
             }
             LifeCycle::AnimFrame(interval) => {
                 self.t += (*interval as f64) * 1e-9;
                 if self.t >= 1.0 {
                     self.t = 0.0;
                 }
                 ctx.request_anim_frame();
-                ctx.request_paint();
```

Comment:
This strikes me as strange - does `AnimFrame` implicitly request a repaint? It seems to me that there are reasonable animations where you wouldn't want that to be the case.

Reply:
Nope, you're right, the new test for the animated spinner was failing because of that.

(Hurray for screenshot tests!)